CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+   (see the ["Building"](#building) chapter).
+1. Contributor submits the PR if the local build is successful and tests are green.
+2. Reviewer put his name to the "Reviewers" section of the proposed PR at the start of the review process.
+3. Reviewer leaves the comments or marks the PR with the word "LGTM."
+4. Contributor answers the comments or fixes the proposed PR.
+5. Reviewer marks the PR with the word "LGTM."
+6. Maintainer could suggest merging the master branch to the PR branch a few times due to changes in the `master` branch.
+7. Maintainer runs TC builds (unit tests and examples as integration tests).
+8. The TC writes the result (passed or not passed) to the PR checks at the bottom of the proposed PR.
+9. If it is possible, maintainers share the details of the failed build with the contributor.
+10. Maintainer merges the PR if all checks are successful and there is no conflict with the master branch.
+
+## How to fix an existing issue
+
+* If you are going to work on the existing issue:
+    * Comment on the existing issue if you want to work on it.
+    * Wait till it is assigned to you by [maintainers](#contacting-maintainers).
+    * Ensure that the issue not only describes a problem, but also describes a solution that had received a positive feedback. Propose a solution if there isn't any.
+* If you are going to submit your first PR in this project:
+    * Find tickets with the label ["good first issue"](https://github.com/Kotlin/dataframe/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee)
+      which are not assigned to somebody.
+    * Learn the [`examples`](https://github.com/Kotlin/dataframe/tree/master/examples) module, submit new interesting example or improve documentation for one of them.
+* If you are ready to participate in library design and in new experiments, find tickets with the label
+  ["research"](https://github.com/Kotlin/dataframe/issues?q=is%3Aissue+is%3Aopen+label%3Aresearch)
+  or join to our [discussions](https://github.com/Kotlin/dataframe/discussions).
+
+## Environment requirements
+
+JDK >= 11 referred to by the `JAVA_HOME` environment variable.
+
+## Building
+
+This library is built with Gradle.
+
+* Run `./gradlew build` to build. It also runs all the tests.
+* Run `./gradlew <module>:test` to test the module you are looking at to speed
+  things up during development.
+
+You can import this project into IDEA, but you have to delegate build actions
+to Gradle (in Preferences -> Build, Execution, Deployment -> Build Tools -> Gradle -> Runner)
+
+## Contacting maintainers
+
+* If something cannot be done, not convenient, or does not work &mdash; submit an [issue](#submitting-issues).
+* To attract attention to the problem or raised question or a new comment, mention one of us on the Github: @koperagen @Jolanrensen @zaleslaw @ileasile
+* "How to do something" questions &mdash; [StackOverflow](https://stackoverflow.com).
```

Comment:
Personally, i never checked SO. There are few questions about our library. I see questions in Slack and issues more frequently